web3/types.py

```diff
@@ -81,6 +81,7 @@
     "blockNumber": int,
 })
 
+JsonRpcResponse = Dict[str, Any]
```

Comment:
Can we do better with a `TypedDict` since we generally know what the fields are for valid `JSON-RPC` responses?